lib/suite-builder.js

```diff
@@ -0,0 +1,154 @@
+'use strict';
```

Comment:
Why moving suite-builder to separate file? This is private part of suite API, it may not be needed anywhere externally.